docs/lvmd.md

```diff
@@ -11,9 +11,10 @@
 
 Command-line options are:
 
-| Option         | Type   | Default value            | Description                                |
-| -------------- | ------ | ------------------------ | ------------------------------------------ |
-| `config`       | string | `/etc/topolvm/lvmd.yaml` | Config file path for device-class settings |
+| Option      | Type   | Default value            | Description                                           |
+| ----------- | ------ | ------------------------ | ----------------------------------------------------- |
+| `config`    | string | `/etc/topolvm/lvmd.yaml` | Config file path for device-class settings            |
+| `container` | -      | not set                  | Switch of toggling whether lvmd runs in the container |
```

Comment:
Could you explain the detail of `container` field?  If the description is correct, "Type" should be bool.

Reply:
fixed